spec/queries/get_embedded_content_spec.rb

```diff
@@ -0,0 +1,130 @@
+RSpec.describe Queries::GetEmbeddedContent do
+  describe "#call" do
+    it "returns data prepared by the presenter" do
+      target_content_id = SecureRandom.uuid
+      allow(Document).to receive(:find_by).with(content_id: target_content_id).and_return(anything)
+
+      presenter_double = double(Presenters::Queries::EmbeddedContentPresenter)
+      expect(Presenters::Queries::EmbeddedContentPresenter).to receive(:new)
+        .with(target_content_id, kind_of(ActiveRecord::Relation))
+        .and_return(presenter_double)
+
+      stubbed_response = {}.to_json
+      expect(presenter_double).to receive(:present).and_return(stubbed_response)
+
+      result = described_class.new(target_content_id).call
+
+      expect(result).to eq(stubbed_response)
+    end
+
+    context "when the target_content_id doesn't match a Document" do
+      it "returns 404" do
+        expect { described_class.new(SecureRandom.uuid).call }.to raise_error(CommandError) do |error|
+          expect(error.code).to eq(404)
+          expect(error.message).to eq("Could not find an edition to get embedded content for")
+        end
+      end
+    end
+
+    context "when the target_content_id does not match a live edition" do
```

Comment:
Is it more accurate to say something like `when the target_content is embedded in non-live editions`...`it does not include the non-live editions in the results`?

I was initially reading this as us searching for the `content_block` via the `target_content_id`, so this might make that a bit clearer.

Reply:
Yes this wasn't worded well was it! As i'm changing the behaviour to include draft and published I'll revamp the tests in a new commit which hopefully makes it clearer 👍